lib/export_dsi_approvers_to_big_query.rb

```diff
@@ -16,28 +16,30 @@ def run!
   def present_for_big_query(batch)
     batch.map do |user|
       {
-        user_id: user["userId"],
-        role_id: user["roleId"],
-        role_name: user["roleName"],
         email: user["email"],
         family_name: user["familyName"],
         given_name: user["givenName"],
+        la_code: la_code(user),
+        trust_uid: user.dig("organisation", "uid"),
+        role_id: user["roleId"],
+        role_name: user["roleName"],
         school_urn: user.dig("organisation", "urn"),
-        organisation_uid: user.dig("organisation", "uid"),
+        user_id: user["userId"],
       }
     end
   end
 
   def insert_table_data(batch)
     dataset.insert TABLE_NAME, present_for_big_query(batch), autocreate: true do |schema|
-      schema.string "user_id", mode: :required
-      schema.string "role_id", mode: :required
-      schema.string "role_name", mode: :required
       schema.string "email", mode: :required
       schema.string "family_name", mode: :required
       schema.string "given_name", mode: :required
+      schema.integer "la_code", mode: :nullable
+      schema.string "role_id", mode: :required
+      schema.string "role_name", mode: :required
       schema.integer "school_urn", mode: :nullable
-      schema.integer "organisation_uid", mode: :nullable
+      schema.integer "trust_uid", mode: :nullable
+      schema.string "user_id", mode: :required
```

Comment:
Changes here:
- sort alphabetically by column
- rename organisation_uid to trust_uid
- add la_code